vnext/Microsoft.ReactNative/Fabric/Composition/CompositionViewComponentView.cpp

```diff
@@ -1305,6 +1305,17 @@ void CompositionBaseComponentView::EnsureTransformMatrixFacade() noexcept {
   }
 }
 
+void CompositionBaseComponentView::updateAccessibilityValue(std::string newValue) noexcept {
```

Comment:
@acoates-ms The value being set in SetValue() goes here and then is modified through a call to UpdateUiaProperty() - which from my understanding also calls the UiaRaiseAutomationPropertyChangedEvent(), updating insights that the property has changed. Do I still need to add a UiaRaiseAutomationEvent() here as well?